tests/integ/test_udf.py

```diff
@@ -1668,22 +1661,17 @@ def run_scikit_fuzzy() -> str:
     IS_IN_STORED_PROC,
     reason="Subprocess calls are not allowed within stored procedures",
 )
-@pytest.mark.skipif(
-    os.getenv("OPENSSL_FIPS") == "1" or IS_WINDOWS,
-    reason="Fasttext will not build on this environment",
-)  # Note that if packages specified are native dependent + unsupported by our Anaconda channel,
-# and users do not have the right gcc setup to locally install them, then they will run into Pip failures.
 def test_add_requirements_with_native_dependency_force_push(session):
     with patch.object(session, "_is_anaconda_terms_acknowledged", lambda: True):
-        session.add_packages(["fasttext"])
+        session.add_packages(["catboost"])
```

Comment:
Catboost does not require a specific GCC compiler and builds fine on Windows/FIPS env, therefore less likely to be flaky.